docs/concepts/05_credentials/02_ctypes.md

```diff
@@ -2,124 +2,131 @@
-- The format field is optionally:
-  - *Date* format e.g., 2012-04-23T18:25:43.511Z
-  - *Time* format e.g., T18:25:43.511Z
-  - *URI* format e.g., "https://www.example.com"
+KILT computes the hash from the following fields of the CType schema:
 
-<CodeBlock className="language-json" title="CType schema example">
-  {ctypeSchema}
-</CodeBlock>
+-   `$schema`
+-   `properties`
+    -   `key`
+    -   `$ref`
+    -   `type`
+    -   `format`
+-   `title`
+-   `type`
 
-The CType schema is afterwards hashed to generate its own identifier, and it becomes the full CType object:
+A typical CType ID looks like this: `kilt:ctype:0xda3861a45e0197f3ca145c2c209f9126e5053fas503e459af4255cf8011d5101`.
 
-<CodeBlock className="language-json" title="Full CType example">
-  {ctype}
-</CodeBlock>
+## Storing and querying CTypes
 
-## CType Metadata
+As of the [KILT runtime 1.9.0][kilt-runtime-1.9.0], you can query CTypes directly from any KILT archive node.
 
-CType Metadata can be linked to a given CType to provide title and descriptions in different languages for the whole CType and its properties.
+After creating a CType, its full content is only included in the blockchain block history and its hash and creation block number anchored to the blockchain state.
 
-<!-- TODO: Add example of CType metadata -->
+To query the full content of a CType, use its hash to look up the creation block number, and use that to query any KILT archive node for the extrinsic information about the CType.
 
-## Hashing
+The returned information includes the whole CType, which is now available for the user to, for example, verify credentials against it.
 
-The hash of the CType is used to identify and anchor it to the KILT blockchain.
+:::info CType creation cost
 
-### Constructing the `hash` for the `$id`
+It costs 0.001 KILT to create a CType on the KILT blockchain.
+
+:::
+
+For a detailed developer-oriented guide to KILT CTypes, read the [CType Cookbook section](../../develop/01_sdk/02_cookbook/04_claiming/01_ctype_creation.md).
 
-KILT uses the hashing algorithm `blake2b256` to compute the hash of CTypes.
-Before hashing, the CType object is sorted by a canonicalization algorithm to ensure that semantically equivalent CTypes with different order of their properties result in the same final hash.
+[kilt-runtime-1.9.0]: https://github.com/KILTprotocol/kilt-node/releases/tag/1.9.0
 
-The hash is computed from the following fields of the CType schema:
+<!-- TODO: Do we still need this? -->
 
-- `$schema`
-- `properties`
-  - `key`
-  - `$ref`
-  - `type`
-  - `format`
-- `title`
-- `type`
+:::danger Deprecation Warning: CType metaschema draft-01
 
-A typical CType ID would look like this: `kilt:ctype:0xda3861a45e0197f3ca145c2c209f9126e5053fas503e459af4255cf8011d5101`.
+CTypes based on the [Draft 01](http://kilt-protocol.org/draft-01/ctype%23%60) metaschema are susceptible to faulty or malicious attester integrations that may introduce unexpected properties to a claim.
```

Comment:
It is the other way around. 

Because you could add additional properties to the credential, you could fool the attesters and get some information attested that they didn't really checked. 

The attester could have checked for the properties they expect and while verifying against the cType schema everything would have looked green. 


Example: 
You could than add this properties to your gym credential: 
```
{ "VIP": true,
 "unlimitedMembership": true}
 ```
And have that attested by the Gym itself. 

Then show to the people at the gym's desk that you don't need to pay to use the facilities and they can verify your credential to proof you are not lying. But in reality you just have a certified lie.